test/integration/migrate/index.js

```diff
@@ -72,6 +72,22 @@ module.exports = function(knex) {
       });
     }
 
+    if (knex.client.driverName === 'sqlite3') {
+      it('should not fail rename-and-drop-column with multiline sql from legacy db', () => {
```

Comment:
Probably sample legacy db that you are using should be only referenced within this test

Reply:
I tried to change the connection and file in config during this case, but I can not figure out how to change change Knex connection config after instantiation. Can you suggest how can I do it?

Reply:
Worst case scenario, you can instantiate a new Knex within this test, it is sqlite-specific anyway so multi-db support doesn't matter here. It is going to be a problem if we ever support alternate sqlite driver, so just to be on a safe side here, I would add `if (knex.client.driverName !== 'sqlite3') throw new Error('Please implement alternate instantiation logic for this driver')'` next to Knex instantiation to protect against someone adding alternate supported driverName blindly.

Reply:
I changed it as you requested, we use this special DB file only in this test. I had to make a copy before the test run and delete it after because knex migration tables changes, so the original file becomes changed in Git for every test run.